test/multiGiveaway/multiGiveaway.test.ts

```diff
@@ -368,6 +372,66 @@ describe('Multi_Giveaway', function () {
       expect(ownerLandId6).to.equal(claim.to);
     });
 
+    // eslint-disable-next-line mocha/no-setup-in-describe
+    [1, 2, 3, 4, 8, 16, 32, 64, 128].forEach((numberOfAssets) => {
+      // eslint-disable-next-line mocha/no-skipped-tests
+      it.skip(`User can claim allocated ${numberOfAssets} tokens from Giveaway contract`, async function () {
```

Comment:
skip ?

Reply:
Maybe do only one batch of 64 and remove the skip

Reply:
Done